Homework 28.09.21/PhoneBook/Tests.c

```diff
@@ -0,0 +1,113 @@
+#include "Tests.h"
+#include "ActionsOnPhoneBook.h"
+#define _CRT_SECURE_NO_WARNINGS
+#include <string.h>
+#include <stdio.h>
+
+bool testForFindNumber()
+{
+    PhoneBook testArray[10];
+    int count = 0;
+    readInitialFile(testArray, &count, "FileTest.txt");
+    char nameForSearch[3][10] =
+    {
+        {'I', 'v', 'a', 'n', '\0'},
+        {'M', 'a', 's', 'h', 'a', '\0'},
+        {'V', 'a', 'n', 'y', 'a', '\0'}
+    };
```

Comment:
Вы не поверите, но
```suggestion
    char nameForSearch[3][10] =
    {
        "Ivan",
        "Masha",
        "Vanya"
    };
```
И это в точности одно и то же, разве что строки нередактируемы (и стоило бы const char тогда)